packages/date-time/src/components/DatePicker/DatePicker.base.tsx

```diff
@@ -418,6 +416,10 @@ export const DatePickerBase = React.forwardRef(
       // don't need to focus the text box, if necessary the focusTrapZone will do it
     };
 
+    const calloutDismissed = (ev: React.MouseEvent<HTMLElement>): void => {
```

Comment:
What's the purpose for this added indirection?

Reply:
looks like my earlier comment didn't get added in time, I'll copy/paste it here:

`unrelated bug fix that was annoying me- onDismiss accepts a function where the first parameter is a mouseEvent, but we were passing a function where the first parameter is an option Date. so the calendarDismissed function would get called with a MouseEvent instead of a Date and we'd get an error in the console`

I'm actually not sure why Typescript didn't catch this- perhaps because both parameters were optional, typescript says both being undefined means it's valid?